betterself/users/urls.py

```diff
@@ -3,34 +3,11 @@
 
 from django.conf.urls import url
 
-from . import views
+from betterself.users import views
 
 urlpatterns = [
-    # URL pattern for the UserListView
-    url(
-        regex=r'^$',
-        view=views.UserListView.as_view(),
-        name='list'
-    ),
-
-    # URL pattern for the UserRedirectView
-    url(
-        regex=r'^~redirect/$',
-        view=views.UserRedirectView.as_view(),
-        name='redirect'
-    ),
-
-    # URL pattern for the UserDetailView
-    url(
-        regex=r'^(?P<username>[\w.@+-]+)/$',
-        view=views.UserDetailView.as_view(),
-        name='detail'
-    ),
-
-    # URL pattern for the UserUpdateView
-    url(
-        regex=r'^~update/$',
-        view=views.UserUpdateView.as_view(),
-        name='update'
-    ),
+    url(regex=r'^$', view=views.UserListView.as_view(), name='list'),
```

Comment:
? look into django docs / not sure what these originally do (maybe from cookiecutter cutover??)